services-api/src/main/java/io/scalecube/services/ExtendedMicroservicesContext.java

```diff
@@ -0,0 +1,21 @@
+package io.scalecube.services;
+
+import io.scalecube.net.Address;
+import io.scalecube.services.discovery.api.ServiceDiscovery;
+
+/**
+ * Context of Scale Cube node. Used in {@link ServiceFactory}. Compared to MicroservicesContext, it
+ * gives access to ServiceDiscovery.
+ *
+ * @see ServiceFactory
+ */
+public interface ExtendedMicroservicesContext extends MicroservicesContext {
```

Comment:
Let's have single one `MicroserviceContext` interface.

Reply:
The smaller the entities, the better. But I don't want to describe the contract in java doc, I think it's not enough. I have another idea to remove MicroservicesContext for ServiceFactory#getServiceDefinition. Because I don't see any cases for using MicroservicesContext